R/query_all_versions.R

```diff
@@ -109,6 +132,12 @@ query_all_versions<- function(node, object_pid, fields = "all"){
     return(df_query)
 }
 
-#Example
-#df<- query_all_versions(mn, "doi:10.18739/A27D2Q670", c("id", "title", "origin", "submitter"))
+
+#' @examples
```

Comment:
move the examples up so it's part of the documentation for `query_all_versions`.  
#' is the multi-line comment character, so we want to keep all the documentation for one function together.